src/vector/_methods.py

```diff
@@ -3285,6 +3291,14 @@ def isclose(
             )
         return isclose.dispatch(rtol, atol, equal_nan, self, other)
 
+    def sum(self) -> VectorProtocol:
+        from vector._compute.planar import sum
+
+        return sum.dispatch(self)
+
+    def count_nonzero(self) -> ScalarCollection:
+        return self.lib.count_nonzero(self.rho2, axis=-1)
```

Comment:
Should this be in a `_compute` module for completeness, or is it acceptable given that we don't need to compute anything new?